store/tx_manager.go

```diff
@@ -270,37 +275,41 @@ func (txm *EthTxManager) GetLINKBalance(address common.Address) (*assets.Link, e
 	return (*assets.Link)(balance), nil
 }
 
-// MeetsMinConfirmations returns true if the given transaction hash has been
+// BumpGasUntilSafe returns true if the given transaction hash has been
 // confirmed on the blockchain.
-func (txm *EthTxManager) MeetsMinConfirmations(hash common.Hash) (bool, error) {
-	if !txm.Connected() {
-		return false, ErrPendingConnection
```

Comment:
I feel like we still need to check if we're connected before moving foward in `BumpGasUntilSafe`. Can you walk through why we no longer need `ErrPendingConnection`?

Reply:
Good catch. I mistakenly thought that was checked under the hood. It wasn't, so I introduced a helper for it instead.